noxfile.py

```diff
@@ -468,8 +474,7 @@ def cover(session):
 @nox.session(python=DEFAULT_PYTHON_VERSION)
 def docs(session):
     """Build the docs for this library."""
-
-    session.install("-e", ".")
+    session.install("-e", ".[scikit-learn]")
```

Comment:
I'm very concerned if we need to install `scikit-learn` just to build the docs. Please remove.

```suggestion
    session.install("-e", ".")
```

Reply:
Not sure if it's possible to remove this as we are calling publish_api_coverage.py, and one of the things it do is go through sklearn APIs and generate a coverage report.

Reply:
Good point. Thanks for clarifying.